### PR TITLE
Fixes #683

### LINKED ISSUE #683: `/next` in querystring should take precedence over default or property settings
Steps to reproduce:

1) Start `Spring WebMVC` example in the Java SDK project

2) Browse to: `http://localhost:8080/login?next=/foo`

3) Enter valid `username` and `password`

4) You will *not* be redirected to `/foo` as you should be

### PR DESCRIPTION
**Pull request by jarias**
Login should redirect to the next query param if exists

### CODE REVIEW

File: extensions/servlet/src/main/java/com/stormpath/sdk/servlet/filter/LoginPageRedirector.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
  */
 public interface LoginPageRedirector {
 
-    public static final LoginPageRedirector INSTANCE = new DefaultLoginPageRedirector();
+    LoginPageRedirector INSTANCE = new DefaultLoginPageRedirector();
```
Comment (dogeared) [inline extensions/servlet/src/main/java/com/stormpath/sdk/servlet/filter/LoginPageRedirector.java]: why remove the `public static final`?

**Comment by dogeared** [comment]
UAT Performed:

* Run the `LoginIT.loginRedirectsToNextQueryParameter` test in the [TCK](https://github.com/stormpath/stormpath-framework-tck) project.

Results:

1) PASS

     Servlet
     Spring WebMVC
     Spring Boot WebMVC

2) FAIL

    Spring Security Spring WebMVC
    Spring Security Spring Boot WebMVC

Failure is due to Spring Security login success handler not being configured to recognize the `next` query parameter.

**Comment by dogeared** [comment]
UAT Performed:

* Run the `LoginIT.loginRedirectsToNextQueryParameter` test in the [TCK](https://github.com/stormpath/stormpath-framework-tck) project on the `more_csrf_and_cookies` branch (which handles CSRF).

Results:

1) PASS

     Servlet
     Spring WebMVC
     Spring Boot WebMVC
     Spring Security Spring WebMVC
     Spring Security Spring Boot WebMVC

File: extensions/servlet/src/main/java/com/stormpath/sdk/servlet/filter/LoginPageRedirector.java (hunk 1 above)
Comment (jarias) [inline extensions/servlet/src/main/java/com/stormpath/sdk/servlet/filter/LoginPageRedirector.java]: All interface fields are by default public static final

Comment (dogeared) [inline extensions/servlet/src/main/java/com/stormpath/sdk/servlet/filter/LoginPageRedirector.java]: hah. right. missed that it was an interface and not a class.

**Pull request merged**